cropharvest/eo/eo.py

```diff
@@ -471,9 +494,6 @@ def export_for_labels(
             )
             if export_started:
                 exports_started += 1
-                if exports_started % 100 == 0:
-                    print(f"{exports_started} exports started")
```

Comment:
Removing this print message, since there is tqdm in the for loop now which helps track progress